src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceFunctions.kt

```diff
@@ -24,10 +25,10 @@ fun Licence.getSentenceChanges(newSentence: UpdateSentenceDatesRequest): Sentenc
     nullableDatesDiffer(newSentence.topupSupervisionStartDate, this.topupSupervisionStartDate)
   val tusedChanged =
     nullableDatesDiffer(newSentence.topupSupervisionExpiryDate, this.topupSupervisionExpiryDate)
+  val prrdChanged = nullableDatesDiffer(newSentence.postRecallReleaseDate, this.postRecallReleaseDate)
+  val isMaterial = (lsdChanged || ledChanged || tussdChanged || tusedChanged || prrdChanged || (sedChanged && this.statusCode == LicenceStatus.APPROVED))
 
-  val isMaterial = (lsdChanged || ledChanged || tussdChanged || tusedChanged || (sedChanged && this.statusCode == LicenceStatus.APPROVED))
-
-  return SentenceChanges(lsdChanged, ledChanged, sedChanged, tussdChanged, tusedChanged, isMaterial)
+  return SentenceChanges(lsdChanged, ledChanged, sedChanged, tussdChanged, tusedChanged, prrdChanged, isMaterial)
```

Comment:
just asked about whether this is needed

Reply:
Shirle verified that we should let them know as we do with other date changes.